check-and-talk/src/components/ProfileEdit.css

```
@@ -0,0 +1,178 @@
+/* color
+back_1 : 464646
+back_2 : 333333
+yellow : FFDC5F
+orange : FFB341
+white : FFFFFF
+white_back : FFFFFF, 0.5
+*/
+
+.ProfileEdit-style {
+    display: flex;
+    width: 381px;
+    height: 520px;
+    border-radius: 20px;
+    border: rgba(255, 255, 255, 0.5);
+    border-width: 5px;
```

```suggestion
    border-width: 2px;
```

border 두께가 조금 두꺼워서 웹상에서 부담스럽게 느껴질 수 있을 것 같아요